utilities/Blog.md

```diff
@@ -0,0 +1,61 @@
+# Insertion Sort Array 
```

Comment:
Fantastic blog! I actually saw this on linkedin when you posted it, randomly came up in my feed.